### ISSUE TITLE
Hello, I have some questions

### ISSUE THREAD

**Issue by 22w2w2** [Hguimaraes/gtzan.keras#2]
Hello, I am very interested in your code. Similarly, I am also a student in the school. Can you support the paper behind this project? If so, I want to read this paper

**Comment by Hguimaraes** [comment]
Hello @22w2w2 !

This code is based in more then one code and also from web posts. The main resource that I used to start this work was: http://benanne.github.io/2014/08/05/spotify-cnns.html
There you can find a link to his phd thesis and articles on this theme.

(Btw, this work here is not finished yet. I'm using it on my undergradute thesis and soon I will pull the final version)

**Comment by stenlytw** [comment]
Hi,

I've run your code and didn't get as good as yours. This is my results:
Validation accuracy - mean: 0.5673333337783812, std: 0.028394052613758283
Test accuracy - mean: 0.5604000000000001, std: 0.027543420266916717
Test accuracy MVS - mean: 0.636, std: 0.04409081537009721

What architectures do you use? Do you tune the model?

Thanks.
